app/views/books/index.html.erb

```diff
@@ -0,0 +1,8 @@
+<h1>Library Catalog</h1>
+
+<%= link_to "Add a new book", new_book_path %>
+
+<ul class="books">
+  <% @books.each do |book| %>
+    <li> Book: <%= book.title %> | Author: <%= book.author %></li>
+<% end %>
```

Comment:
The end should be indented two spaces so it lines up with the beginning of the `<% @books.each` block. Also you'll want to add a closing `</ul>` tag